### PR TITLE
Improve readability in a few lines for babel 7 migration doc.

### PR DESCRIPTION
**Pull request by obedparla**
* Fix a typo or two.
* Add a few commas
* Make the text in a few lines more readable.

### CODE REVIEW

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit 7a7cc3801616ff3349bda6693032f615a1d183b0

https://deploy-preview-1636--babel-new.netlify.com

File: docs/v7-migration.md [hunk 1]
```diff
@@ -28,14 +28,14 @@ The "env" preset has been out for more than a year now, and completely replaces
 - `babel-preset-es2016`
 - `babel-preset-es2017`
 - `babel-preset-latest`
-- can combination of the above ^
+- A combination of the above ^
 
-We suggest using switching as those presets should be able to be substituted entirely with the env preset.
+These presets can be substituted with the "env" preset.
```
Comment (nicolo-ribaudo) [inline docs/v7-migration.md]: It's more than "can", we are actively encouraging it.

Comment (hzoo) [reply docs/v7-migration.md]: Yeah I guess it's a "should", don't think that's too opinionated because we deprecated the other ones

**Comment by hzoo** [comment]
awesome thanks!

**Comment by obedparla** [comment]
:D

**Pull request merged**